doc/controls/DrawerFlyoutPresenter.md

```diff
@@ -57,3 +102,34 @@ Example:
 </Button>
 ```
 > note: Here `VisibleBoundsPadding.PaddingMask` is used to prevent the content from being placed outside of the user-interactable area on mobile devices.
+
+### Extended Use Cases
+- Rounded Corner
+    ```xml
+    <Flyout Placement="Full">
+        <Flyout.FlyoutPresenterStyle>
+            <Style BasedOn="{StaticResource BottomDrawerFlyoutPresenterStyle}" TargetType="FlyoutPresenter">
+                <Setter Property="CornerRadius" Value="16,16,0,0" />
+            </Style>
+        </Flyout.FlyoutPresenterStyle>
+        <Border toolkit:VisibleBoundsPadding.PaddingMask="All" Padding="16,16,0,0">
```

Comment:
Why `toolkit:VisibleBoundsPadding` and not `utu:SafeArea`?

Reply:
This is the legacy & tested way. I will probably update these in another PR separate from this.